app/models/external_activity.rb

```diff
@@ -209,7 +199,7 @@ def full_title
   end
 
   def activities
-    template.activities if material_type == 'Investigation'
+    template.activities if material_type == 'Investigation' && !template.nil?
```

Comment:
I'd switch this to `template_type == 'Investigation'`.  
In theory an author could publish a LARA Activity, and then change its material_type to be 'Investigation'. That would be weird, but possible. So it seems good for this method to handle that case without breaking.